notices.txt

```diff
@@ -1,109 +1,202 @@
+-------------------------------------------------------------------------
+
+Contents of NOTICE text file for NativeScript Doctor v0.9*:
+=========================================================================
+==  NOTICE file corresponding to section 4 d of            	 	  ==
+==  the Apache License, Version 2.0,                                   ==
+==  in this case for NativeScript Doctor v0.9					  ==
```

Comment:
we have plans to update nativescript-doctor to 0.12.0 - is it a problem?

Reply:
It will not be a problem. I will update it when changed.